sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/ResourceConfigurationTest.java

```diff
@@ -46,6 +49,27 @@ void emptyClassLoader() {
                 (r, c) -> r)
             .getAttributes();
 
+    assertProcessAttributeIsNull(attributes);
+  }
+
+  @Test
+  void onlyEnabledCustomResourceProvider() {
```

Comment:
I think it would be good to have a unit test to verify the behavior when things are specified in both lists, especially when the same provider is in both lists.

Reply:
Sounds reasonable, I will add the missing unit tests.